README.md

```diff
@@ -1,11 +1,11 @@
 <img width="200" align="right" src="img/uhyve.svg" />
 
-# Uhyve - A minimal hypervisor for Hermit
```

Comment:
I actually like the tagline. Why should we remove it?

Reply:
My personal opinion is that it looks slightly because of the line wrap, and its content is not much different to what is said right below the header. I think this would look better (the icons were put under the tagline, this PR doesn't have that).

![image](https://github.com/user-attachments/assets/4f0432d3-37ac-4740-9c40-44470f23ec54)

Nevertheless, it's a matter of preference and not a hill to die on.